### DISCUSSION TITLE
Middle align horizontal stack

### DISCUSSION THREAD

**Discussion by jhanley87** [twilio-labs/paste#2796] [Q&A] [answered]
Hey all using the <Stack> in horizontal to layout a name with Avatar, is there an easy way to align both to the middle? at the moment it looks like its aligning to the top.

![image](https://user-images.githubusercontent.com/96482686/199487981-add0f6b2-0eab-4bdd-aa55-da86bbe925bc.png)

code I am using at the moment
```
<Stack orientation={"horizontal"} spacing={"space30"}>
              <Avatar
                name="customer avatar"
                size={"sizeIcon110"}
                src="https://ui-avatars.com/api/?name=John+Doe"
              ></Avatar>
              <Heading
                variant="heading10"
                as="h1"
              >John Doe</Heading>
</Stack>
```

**Comment by SiTaggart** [comment, marked as answer]
Hey

Adding `marginBottom="space0"` to the heading component should do the trick for you.

Let us know.

**Reply by TheSisb** [reply]
You may also want to consider our [Media Object](https://paste.twilio.design/components/media-object) component for this UI.

**Comment by jhanley87** [comment]
These both worked, thankyou... amazing :-)
